components/slack/join-form.js

```diff
@@ -79,19 +82,28 @@ const JoinForm = ({ sx = {} }) => {
             required
           />
         </Label>
+        {isAdult && (
+          <Text
+            variant="caption"
+            color="secondary">
+              Hold your horses! Our Slack community for teenagers, and as such
+              we're really careful about letting adults join. If you feel you'd
+              have a place here, reach out to conduct@hackclub.com.
```

Comment:
```suggestion
              have a place here, reach out to <Link href="mailto:conduct@hackclub.com">conduct@hackclub.com</Link>.
```

Reply:
It feels a bit weird to have people reaching out to conduct@hackclub.com, "conduct" implies they've done something wrong